src/test/java/com/notably/logic/suggestion/block/BlockTreeStub.java

```diff
@@ -0,0 +1,38 @@
+package com.notably.logic.suggestion.block;
+
+import com.notably.commons.core.path.AbsolutePath;
+import com.notably.model.block.Block;
+import com.notably.model.block.BlockTree;
+import com.notably.model.block.BlockTreeItem;
+
+/**
+ * A Stub class for BlockTree.
+ */
+public class BlockTreeStub implements BlockTree {
```

Comment:
Similarly, I don't think you need to create stubs for `BlockTree`. Firzan has created `BlockTreeImpl`, which you can instantiate directly!